README.md

```diff
@@ -330,6 +331,9 @@ Focus is on the specific task.
 - [Rocketeer](https://github.com/Anahkiasen/rocketeer) - Deployment
 - [Travis CI](https://travis-ci.org/) - A continuous integration platform - de-facto standard for testing (GitHub) repos.
 
+## Feature Toggle
+- [Swivel](https://github.com/zumba/swivel-cake) - A library that allows to setup up to 10 different behaviors with an interface for tracking results.
```

Comment:
@dereuromark Yes, it is a cake plugin that wraps the functionality of the standalone library (swivel).